js/src/utils/urls.js

```diff
@@ -58,9 +59,17 @@ export const getEditStoreAddressUrl = () => {
 	);
 };
 
-export const getReconnectAccountsUrl = () => {
+export const getReconnectWPComAccountUrl = () => {
 	return getNewPath(
-		{ subpath: subpaths.reconnectAccounts },
+		{ subpath: subpaths.reconnectWPComAccount },
+		settingsPath,
+		null
+	);
+};
+
+export const getReconnectGoogleAccountUrl = () => {
+	return getNewPath(
+		{ subpath: subpaths.reconnectGoogleAccount },
 		settingsPath,
 		null
 	);
```

Comment:
💅 

Related to https://github.com/woocommerce/google-listings-and-ads/pull/1385/files#r839576276

WDYT of 

```
export const getReconnectAccountUrl = ( accountType ) => {
let subpath = subpaths.reconnectWPComAccount;

if (accountType === GOOGLE_ACCOUNT) {
   subpath = subpaths.reconnectGoogleAccount;
}

return getNewPath(
		{ subpath },
		settingsPath,
		null
	);

}

```

Reply:
Adjusted in https://github.com/woocommerce/google-listings-and-ads/commit/9eb735608b1945a2d546d9421cdcf1d426211923.